### DISCUSSION TITLE
Why does Github not allow non-org accounts to make multiple forks ?

### DISCUSSION THREAD

**Discussion by TheCarpetMerchant** [community/community#69135] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

It is trivial to create an organization, and have multiple forks of the same project in it. Why not allow the same thing for regular accounts ?

**Comment by Sternbach-Software** [comment]
Use case: I am porting a library to another language (where master is in the new language), but to remain compliant as a port, I must maintain feature parity. In order to add a new feature, I need to submit a PR upstream. In order to submit a PR upstream, I need a working dev env of the previous language. It is inconvenient, unorganized, and (debatably) nonsensical to have upstream as a branch of downstream, so I would like to have a separate fork dedicated to upstream PRs.

**Reply by jsoref** [reply]
I personally would create an org for most of this work. Using the personal entity for development work has a lot of downsides...

Currently there's no interface to `hosted-runners` for personal repositories, so I don't have an equivalent of: https://github.com/organizations/check-spelling-sandbox/settings/actions/hosted-runners
<img width="953" alt="image" src="https://github.com/user-attachments/assets/bd7f1bc6-17f8-482c-a481-97e929ceb511" />

There are runner limits per owner, so if you do a lot of work in your personal thing, you're going to suffer...

The permissions model for personal repositories is much more constrained...

I have a number of orgs and my model is:
upstream -> personal -> working-org

I make a personal fork so that when I'm ready to make a PR to upstream, I can, and people can have reasonable faith that random people won't write to it, but I do all of my work in the working-org fork(s).
